regression-test/suites/nereids_syntax_p0/function.groovy

```diff
@@ -103,5 +103,10 @@ suite("nereids_function") {
         sql "select convert_to('abc', cast(number as varchar)) from numbers('number'='1')"
         exception "must be a constant"
     }
+
+    test {
+        sql """select "1" == "123", "%%" == "%%" """
```

Comment:
nit: we need remove syntax `1 == 3`, legacy planner only support `1 = 3`, please fix it in another PR